docs/stock/heikinashi.md

```diff
@@ -0,0 +1,10 @@
+Candlestick chart
+================
+
+A Heiken Ashi series is typically used to present the modified open, high, low and close price over a period of time. 
+A Heiken Ashi chart is composed of a body and an upper and a lower wick similarly to the candlestick. The only difference is that the open, high, low and close values had been modified by the formula.
```

Comment:
> A Heiken Ashi chart

Should we use a different word than "chart"? This is an item or a point, right?

.. _have_ been modified by the formula.